islandora_checksum_checker.drush.inc

```diff
@@ -46,25 +49,75 @@ function islandora_checksum_checker_drush_command() {
     'examples' => array(
       'Standard example' => 'drush --user=fedoraAdmin run-islandora-checksum-queue',
       'Alias example' => 'drush --user=fedoraAdmin ricq',
+      'Options example' => 'drush --user=fedoraAdmin --days-to-complete=180 --cmd-run-frequency=12 run-islandora-checksum-queue',
     ),
     'aliases' => array('ricq'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_LOGIN,
+    'options' => array(
+      'days-to-complete' => array(
+        'description' => 'The number of days it should take to check all the checksums.',
+        'example-value' => 365,
+        'value' => required,
+      ),
+      'cmd-run-frequency' => array(
+        'description' => 'The number of hours between checksum runs (required if --days-to-complete is set).',
+        'example-value' => 12,
+        'value' => required,
```

Comment:
@jordandukart i have a pull in some of my local branches for this. Today is a code heavy release day so will do it once i get out of work.